lib/attackMonitoring.js

```diff
@@ -10,6 +10,9 @@ let iscbFunctionDefined = false; //Default, not defined. When true, can change c
 let cbFuncitonLine = "at TLSSocket.obj.<computed> [as connect] (/mnt/c/Ironwasp/Product/NodeSecurityShield/lib/hook.js:22:13)"
 //Error.stackTraceLimit = 60
 var reportUriHosts = [];
+var violations = 0;
+var violationLimit = 0;
```

Comment:
Set the default violation count value (100) here, this would be easier to identify this value during a review.